lib/takeaway.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'menu'
+require_relative 'twiliosms'
+
+class Takeaway
+  INITIAL_COST = 0.00
+
+  def initialize(menu_object = Menu.new)
+    @menu_object = menu_object
+    @choice = []
+  end
+
+  def menu
+    @menu_object.list
+  end
+
+  def add_item(name, cost)
+    @menu_object.add(name, cost)
+  end
```

Comment:
Should the user of the programme really have access to adding items to the menu? It maybe be better to not have this in here.